graphql_api/views.py

```diff
@@ -201,7 +202,11 @@ def get_validation_rules(
                 maximum_cost=settings.GRAPHQL_QUERY_COST_THRESHOLD,
                 default_cost=1,
                 variables=data.get("variables"),
-            )
+            ),
+            create_max_depth_rule(max_depth=getattr(settings, "GRAPHQL_MAX_DEPTH", 15)),
```

Comment:
Yeah, my intention was to force defensive programming there as I feel like I've seen a lot of accessing of dicts and tribal knowledge on when it's a bug.
But thinking about it again, it does seem like burying a default at this level is the wrong practice and since settings is something we control, we would prefer it to fail early and actually throw an error instead of having it "be forgiving" here.
Fixed it!

Reply:
Haha yep totally agreed! Thanks for making the update